tests/PersonBearbeiten.spec.ts

```diff
@@ -149,7 +149,7 @@ test.describe(`Testfälle für die Administration von Personen": Umgebung: ${pro
         const sperrDatumAb = await generateDateToday() // Konkrete Testdaten für diesen Testfall
 
         await test.step(`Testdaten: Lehrer mit einer Rolle(LEHR) und SP(email) über die api anlegen ${ADMIN}`, async () => {
-          userInfoLehrer = await createRolleAndPersonWithUserContext(page, testschule, typelehrer, await generateNachname(), await generateVorname(), await getSPId(page, email), await generateRolleName());
+          userInfoLehrer = await createRolleAndPersonWithUserContext(page, testschule, typelehrer, await generateNachname(), await generateVorname(), [await getSPId(page, email)], await generateRolleName());
```

Comment:
An anderen Stellen erstellst du hier eine Konstante für die SPIds, hier und etwas weiter unten nicht. Hat das einen bestimmten Grund?

Reply:
Ich finde, dass man die Konstante nicht extra definieren muss, wenn die nur einmal in dem Test verwendet wird.
Wenn das woanders nicht so ist, sind das Altlasten. Ich habe die beiden Stellen korrigiert, so dass die Konstante nicht extra definiert wird.